app/src/main/java/com/breadwallet/entities/IntroLanguageResource.kt

```diff
@@ -115,4 +115,8 @@ class IntroLanguageResource {
             )
         )
     }
+
+    fun findIndex(language: Language): Int {
+        return loadResources().map { it.lang }.indexOf(language)
```

Comment:
Let's add even more description in the fun name:
`findLanguageIndex`
or
`selectLanguageIndex`

or similar so that when we are searching the code I can search with _Language_ or something similar

Also please as the mapping filter ( `it` )to something more descriptive.  `introduction` or `intro`